### PR TITLE
refactor: [M3-7547] - Graphs stories v7 migration

### PR DESCRIPTION
**Pull request by abailly-akamai**
## Description 📝
PR to migrate our Graph stories to v7

## Changes  🔄
- Migrate GaugePercent and add test
- Migrate LineGraph and add test
- Update storybook to `7.6.4`
- Add `@storybook/addon-storysource` to show source code to individual stories

## Preview 📷
Screenshot 2023-12-14 at 16 08 06

Screenshot 2023-12-14 at 16 08 39

### Verification steps 
(How to verify changes)
- run `yarn storybook`
- verify http://localhost:6006/?path=/docs/components-graphs-gaugespercent--documentation
- verify http://localhost:6006/?path=/docs/components-graphs-linegraph--documentation
- run `yarn test GaugePercent.test.tsx`
- run `yarn test /LineGraph.test.tsx`

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [x] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/.storybook/main.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ const config: StorybookConfig = {
     '@storybook/addon-measure',
     '@storybook/addon-actions',
     'storybook-dark-mode',
+    '@storybook/addon-storysource',
```
Comment (abailly-akamai) [inline packages/manager/.storybook/main.ts:16]: There is a code panel for stories in the documentation, but not for individual stories so this add-on helps with that

Screenshot 2023-12-14 at 16 12 25

File: packages/manager/src/components/GaugePercent/GaugePercent.test.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        grey2: '#f4f4f4',
+      },
+      palette: {
+        primary: {
+          main: '#3683dc',
+        },
+      },
+    }),
+  };
+});
```
Comment (bnussman-akamai) [inline packages/manager/src/components/GaugePercent/GaugePercent.test.tsx:22]: Are there any maintainability concerns with this?

**Review by bnussman-akamai — COMMENTED** [review]
Stories and Storybook update looks good! 

I do worry about the amount of vi.mocks and I don't completely understand why they are all needed.

Keep in mind Recharts is getting introduced and we'll hopefully be deleting `LineGraph` and maybe the GaugePercent soon

**Comment by abailly-akamai** [comment]
@bnussman-akamai yeah that's fair. I simplified the tests since they will go away for those specific components.

**Review by bnussman-akamai — APPROVED** [review]
🧼📈

File: packages/manager/src/components/LineGraph/LineGraph.tsx [hunk 3]
```diff
@@ -37,33 +37,74 @@ setUpCharts();
...
    */
   accessibleDataTable?: {
     unit: string;
   };
+  /**
+   * The accessible aria-label for the chart.
+   */
   ariaLabel?: string;
+  /**
+   * The height in `px` of the chart\n`number`.
```
Comment (coliu-akamai) [inline packages/manager/src/components/LineGraph/LineGraph.tsx:62]: seeing a random `\n` appear in the documentation for this and some other props:
image
image

**Review by coliu-akamai — APPROVED** [review]
✅ tests
✅ stories look good!

one small comment below

**Pull request merged**